docs: add translation for owners (#410)
ti-community-infra/tichi: docs/en/plugins/owners.md

  - maintainers
  - techLeaders
  - coLeaders
  - committers
  - reviewers

In the TiDB community collaboration process, **a PR is usually reviewed several times before it is merged**. Therefore, it is important to define the number of `LGTM` required for each PR in this service. You can also specify the label prefix for the number of `LGTM` required by the PR in the configuration. owners will automatically read the required `LGTM` number from the label.

## Design

In order to implement the division of permissions described above, we decided to adopt the RESTFUL interface to define the permissions for each PR.

API: `/repos/:org/:repo/pulls/:number/owners`

The owners will look for labels starting with `sig/` in the current PR and then look for information about the SIG. Finally, owners are generated based on the information of the SIG obtained.

However, there are some special cases where the corresponding SIG cannot be found:
- Some modules do not have clear SIG affiliations at the moment: reviewers and committers using all SIGs of the TiDB community
- Some small repositories belong directly to a SIG: Support for configuring a default SIG for this repository

> - Some repositories have SIG-independent PR: support for using GitHub permissions for repositories to include collaborators with write and admin as reviewers and committers (only works if there is no SIG tag)

Reviewer: ```suggestion
- Some repositories whose PR is independent of SIG: support for using GitHub permissions for repositories to include collaborators with write and admin permissions as reviewers and committers (only works if there is no SIG label)
```
